### ISSUE TITLE
Breaker switch issue(?)

### ISSUE THREAD

**Issue by theunkn0wn1** [BluSunrize/ImmersiveEngineering#442]
I have a test setup where i have the max power throughput of three HV lines feeding into one Hv cable. This cable then runs through a breaker switch before reaching the RF consumer. This setup is shown below.

![image](https://cloud.githubusercontent.com/assets/3110986/9982257/6ab63dd0-5f8a-11e5-9883-3c222fa6b040.png)
 Upon closing the breaker the wire fries, as expected, though the breaker returns to the 'open' position. therein lies the problem i bring up. If the Breaker is acting as a circuit breaker, which it appears to be attempting, then only the right side (the input side) of the circuit should have been fried. If not, then why does such a behavior exist?

Tl;Dr : Breaker switch acts as a breaker, though does not protect the circuit from power surges/overloads
---------------------------

If the proper breaker functionality is not completely implemented yet, i haven't found any documentation stating so. 

===============
Ie version 0.6.0_pre SHA 641c7430f1417b9026e0b74ba5f3f5dcb67a870e

**Comment by BluSunrize** [comment]
They don't act as circuit breakers. I'll try to replicate the issue

**Comment by cobra** [comment]
@theunkn0wn1 s/right/left/

For the most realistic behaviour, the switch should stay closed, but I don't really care about this.
A wire-protecting circuit breaker would be nice nonetheless. New device?

**Comment by theunkn0wn1** [comment]
Ok, Then whilst we are on the subject.

Request for feature: circuit breakers
----------------
The circuit breaker would protect the circuit from pwoer overloads, the input end that contains the fault would burn. but not the output end. The device would have to be rated for a given voltage, so we would have three tiers of breakers. Unpin 'tripping' the breaker would void the current Rf push that triggered it, open the circuit, and wait for the user to reset it. Once opened, it acts like the current breaker switch, breaking the circuit.

 A new device would be well fit for this mod, and is one of the things talked about in #362

**Comment by XFactHD** [comment]
I think the circuit breaker shouldn't be electronic (resettable), it should be electric (with a glass fuse). It would need a fancy renderer (kinda like the electrodes from the arc furnace) and you would need to replace by right clicking to burnt fuse out of the fuse holder and the new one into it.

**Comment by BluSunrize** [comment]
Slow down fellas. This is still an issue on wrongly burning wires. Circuit Breakers are not confirmed and not part of this one.
Let's keep this to the issue at hand

**Comment by theunkn0wn1** [comment]
@BluSunrize the original issue is unexpected behavior with the breaker switch, not cable burn directly. Now that you have confirmed that the Breaker switch is not acting as expected :disappointed: , I have proposed the addition of an entity seemingly forgotten from #362.

**Comment by malte0811** [comment]
@BluSunrize this is caused by the EnergyNet sending block updates with -1 as the id (the first number) when changing connections ([code](https://github.com/BluSunrize/ImmersiveEngineering/blob/e0842f16a2fd01aca27e7c5ff25f5c54950c3e9f/src/main/java/blusunrize/immersiveengineering/api/energy/ImmersiveNetHandler.java#L97)). The breaker switch assumes that an update with an id that is not 1 means that it is open ([code](https://github.com/BluSunrize/ImmersiveEngineering/blob/0e9a224500c6ec7dce16618da2407f82c6620c19/src/main/java/blusunrize/immersiveengineering/common/blocks/metal/TileEntityBreakerSwitch.java#L140)). That line should be changed to something like `this.active = id==1?true:(id==0?false:this.active);`, that fixes it.

**Comment by BluSunrize** [comment]
Oooh right,, yeah, my update thingy. Forgot about that.
